--- vimpager/Makefile	2015-05-07 10:48:56.449685259 +0500
+++ vimpager/Makefile	2015-05-07 10:49:41.870686509 +0500
@@ -8,7 +8,7 @@
 INSTALLCONF?= ${INSTALL} -m 644
 COPYFILE?= ${INSTALL} -m 644
 
-all: vimpager vimpager.1 vimcat.1 README README.md
+all: vimpager
 
 vimpager: ansiesc.tar.uu less.vim.uu perldoc.vim.uu vimcat.uu ConcealRetab.vim.uu
 	mv vimpager vimpager.work
@@ -118,17 +118,4 @@
 
 man: vimpager.1 vimcat.1
 
-%.1: %.md
-	pandoc -s -w man $< -o $@
-	tr -d '\015' < $@ > $@.tmp
-	mv $@.tmp $@
-
-README: vimpager.md
-	pandoc -s -w plain vimpager.md -o README
-	tr -d '\015' < README > README.tmp
-	mv README.tmp README
-
-README.md: vimpager.md
-	${COPYFILE} vimpager.md README.md
-
 .PHONY: all install uninstall man
