--- a/modules/mqtt/mqtt.c	2019-03-14 13:56:26.303106238 +0500
+++ b/modules/mqtt/mqtt.c	2019-03-14 13:58:10.075489296 +0500
@@ -103,7 +103,7 @@
 		broker_host, broker_port, mqttclientid, mqttbasetopic);
 
 	snprintf(mqttsubscribetopic, sizeof(mqttsubscribetopic),
-		"/%s/command/+", mqttbasetopic);
+		"/%s/command", mqttbasetopic);
 	snprintf(mqttpublishtopic, sizeof(mqttpublishtopic), "/%s/event",
 		mqttbasetopic);
 	info("mqtt: Publishing on %s, subscribing to %s\n",
