Index: tdesktop/Telegram/Resources/telegram.qrc
===================================================================
--- tdesktop.orig/Telegram/Resources/telegram.qrc
+++ tdesktop/Telegram/Resources/telegram.qrc
@@ -36,6 +36,7 @@
         <file>qmime/freedesktop.org.xml</file>
     </qresource>
     <qresource prefix="/langs">
+        <file alias="lang_ru.strings">langs/lang_ru.strings</file>
         <file alias="lang_it.strings">langs/lang_it.strings</file>
         <file alias="lang_es.strings">langs/lang_es.strings</file>
         <file alias="lang_de.strings">langs/lang_de.strings</file>
Index: tdesktop/Telegram/SourceFiles/lang.h
===================================================================
--- tdesktop.orig/Telegram/SourceFiles/lang.h
+++ tdesktop/Telegram/SourceFiles/lang.h
@@ -22,6 +22,7 @@ Copyright (c) 2014-2016 John Preston, ht
 
 constexpr const str_const LanguageCodes[] = {
 	"en",
+	"ru",
 	"it",
 	"es",
 	"de",
Index: tdesktop/Telegram/Telegram.pro
===================================================================
--- tdesktop.orig/Telegram/Telegram.pro
+++ tdesktop/Telegram/Telegram.pro
@@ -401,6 +401,7 @@ OTHER_FILES += \
     ./Resources/basic.style \
     ./Resources/all_files.style \
     ./Resources/langs/lang.strings \
+    ./Resources/langs/lang_ru.strings \
     ./Resources/langs/lang_it.strings \
     ./Resources/langs/lang_es.strings \
     ./Resources/langs/lang_de.strings \
