diff -Naur a/Telegram/SourceFiles/lang.h b/Telegram/SourceFiles/lang.h
--- a/Telegram/SourceFiles/lang.h	2016-02-26 18:10:57.000000000 +0500
+++ b/Telegram/SourceFiles/lang.h	2016-03-02 15:52:27.615116142 +0500
@@ -22,6 +22,7 @@
 
 static const char *LanguageCodes[] = {
 	"en",
+	"ru",
 	"it",
 	"es",
 	"de",
diff -Naur a/Telegram/SourceFiles/telegram.qrc b/Telegram/SourceFiles/telegram.qrc
--- a/Telegram/SourceFiles/telegram.qrc	2016-02-26 18:10:57.000000000 +0500
+++ b/Telegram/SourceFiles/telegram.qrc	2016-03-02 15:52:12.327116102 +0500
@@ -36,6 +36,7 @@
         <file>qmime/freedesktop.org.xml</file>
     </qresource>
     <qresource prefix="/langs">
+        <file alias="lang_ru.strings">langs/lang_ru.strings</file>
         <file alias="lang_it.strings">langs/lang_it.strings</file>
         <file alias="lang_es.strings">langs/lang_es.strings</file>
         <file alias="lang_de.strings">langs/lang_de.strings</file>
